libcontainer/seccomp/config.go

```diff
@@ -51,3 +51,33 @@ func ConvertStringToAction(in string) (configs.Action, error) {
 		return 0, fmt.Errorf("string %s is not a valid action for seccomp", in)
 	}
 }
+
+// ConvertStringToArch converts a string into a Seccomp comparison arch.
+func ConvertStringToArch(in string) (string, error) {
```

Comment:
Couldn't this have been a map? I guess it does not matter really but usually maps are a little cleaner.